corehq/apps/hqadmin/views/users.py

```diff
@@ -97,12 +97,22 @@ def post(self, request, *args, **kwargs):
             for user in users:
                 # save user object only if needed and just once
                 if user.is_superuser is not is_superuser:
+                    update_log = {
```

Comment:
@dannyroberts This fulfills our requirements mentioned in the ticket. Just want to sure of the format, if you feel we can log it in a better way?